recipes/xmatchview/meta.yaml

```diff
@@ -0,0 +1,30 @@
+
+package:
+    name: xmatchview
+    version: v1.1.1
+
+source:
+  url: https://github.com/bcgsc/xmatchview/archive/v1.1.1.tar.gz
+  md5: a1fb9eb41ba0d8c0a251f99433de1b7c
+
+build:
+    noarch: python
+    number: 0
+
+requirements:
+    build:
+        - python 2.7
+        - PIL
+
+    run:
+        - python 2.7
+        - PIL
+
+test:
+    imports:
+       - PIL
```

Comment:
Shouldn't this import `xmatchview`?

Reply:
xmatchview.py will be used as a standalone script (https://github.com/bcgsc/xmatchview/blob/dev/setup.py#L31), so it doesn't get imported.

Reply:
In that case, use 

```
test:
  commands:
    - xmatchview -x alig.fa ...
```

to run a test. You can have extra files in the recipe folder for this, just keep them small.

You can publish this here with just a `xmatchview --help` type test case. However, it would be much better if you could have something tiny that makes `xmatchview` run through a basic demo, actually using PIL and some fonts. If you look at e.g. `fastqc` and the trouble we've had with that recipe (missing fonts type stuff), having a test just makes things much safer, in particular since the xmatchview project is active and we might have future releases to double check.